lib/findExports.js

```diff
@@ -0,0 +1,203 @@
+import fs from 'fs';
+import path from 'path';
+
+import parse from './parse';
+import requireResolve from './requireResolve';
+
+function findESNamedExports(node) {
+  if (node.type !== 'ExportNamedDeclaration') {
```

Comment:
I should have done them while I was adding them... Some of them I can't fully recall why they're there. Any addition touching the ast has been very TDD, so hopefully there are good info if you start git-blaming.